pom.xml

```diff
@@ -835,7 +835,7 @@
                     <exclude>com.puppycrawl.tools.checkstyle.gui.ParseTreeTableModel*</exclude>
                     <exclude>com.puppycrawl.tools.checkstyle.gui.TreeTableCellRenderer*</exclude>
                     <exclude>com.puppycrawl.tools.checkstyle.gui.TreeTableModelAdapter*</exclude>
-                    <!-- Deprecated classes -->
+                    <!-- suppress XmlDeprecatedElement Deprecated classes -->
```

Comment:
from inspection description:
`This inspection checks for deprecated XML elements. The elements can be marked by XML comment or documentation tag with text "deprecated".`

new comment looks very weird. If inspection do not like just a word "deprecated", 
lets make comment `<!-- This class in is deprecation phase  -->`

Reply:
Done.